linode/manager
# M3-5605: Update storybook for entity landing page error state

## Description 📝

Converts the `ErrorState` story to `MDX` format. Adds copy and descriptions for props.

## How to test 🧪

1. run `yarn storybook`
2. Navigate to the [story for ErrorState](http://localhost:6006/?path=/docs/features-entity-landing-page-error-display--error-display)
3. Ensure the story renders correctly
